articles/user-guide/host-programs.md

```diff
@@ -341,6 +341,68 @@ Multiple qubits:
 {'CNOT': 0, 'QubitClifford': 4, 'R': 0, 'Measure': 4, 'T': 0, 'Depth': 0, 'Width': 4, 'BorrowedWidth': 0}
 ```
 
+#### Using Q# code from other projects or packages
+
+By default, the `import qsharp` command loads all of the `.qs` files in the current folder and makes their Q# operations and functions
+available for use from inside the Python script.
+
+To load Q# code from another folder, the [`qsharp.projects` API](https://docs.microsoft.com/python/qsharp/qsharp.projects.projects)
+can be used to add a reference to a `.csproj` file for a Q# project (that is, a project that references `Microsoft.Quantum.Sdk`).
+This command will compile any `.qs` files in the folder containing the `.csproj` (and subfolders). It will also recursively load
```

Comment:
I think making  that a parenthetical could be a bit confusing, in that it could imply that it's an optional detail?
```suggestion
This command will compile any `.qs` files in the folder containing the `.csproj` and its subfolders. It will also recursively load
```